src/ServiceControl.Audit.AcceptanceTests/Auditing/When_running_setup_command__integration_test.cs

```diff
@@ -0,0 +1,300 @@
+                {
+                    Directory.Delete(path);
+                }
+            }
+            finally
+            {
+                emptyTempDirectory?.Delete();
+            }
+        }
+        #endregion TearDown
+
+        #region Test helpers
+        public class SqlLocalDb
+        {
+            public static SqlLocalDb CreateNewIn(string path)
+            {
+                return new SqlLocalDb(path);
+            }
+            SqlLocalDb(string path)
+            {
+                _name = $"Test_{Path.GetRandomFileName().Replace(".", string.Empty)}";
+
+                var connection = NewConnectionToMaster();
+                using (connection)
+                {
+                    connection.Open();
+
+                    var sql = $@"CREATE DATABASE[{_name}]ON PRIMARY (NAME={_name}, FILENAME = '{path}\{_name}.mdf')
+                                            LOG ON (NAME={_name}_log, FILENAME = '{path}\{_name}_sql_transport_log.ldf')";
+
+                    var command = new SqlCommand(sql, connection);
+                    command.ExecuteNonQuery();
+                }
+            }
+
+            public void Detach()
+            {
+                using (var connection = NewConnectionToMaster())
+                {
+                    connection.Open();
+                    ExecuteSql(connection, $"ALTER DATABASE [{_name}] SET OFFLINE WITH ROLLBACK IMMEDIATE");
+                    ExecuteSql(connection, $"exec sp_detach_db '{_name}'");
+                }
+
+            }
+
+            static SqlConnection NewConnectionToMaster() => new SqlConnection(@"Data Source=(LocalDB)\MSSQLLocalDB;Initial Catalog=master;Integrated Security=True;Connection Timeout=300;");
+
+            SqlConnection NewConnection()
+            {
+                return new SqlConnection($@"Data Source=(LocalDB)\MSSQLLocalDB;Initial Catalog={_name};Integrated Security=True;Connection Timeout=330;");
+            }
+
+            public int ExecuteSql(IDbConnection connection, string sql)
+            {
+                using (var command = connection.CreateCommand())
+                {
+                    command.CommandText = sql;
+                    command.CommandType = CommandType.Text;
+                    return command.ExecuteNonQuery();
+                }
+            }
+
+            public int ExecuteScalar(IDbConnection connection, string sql)
+            {
+                using (var command = connection.CreateCommand())
+                {
+                    command.CommandText = sql;
+                    command.CommandType = CommandType.Text;
+                    return (int)command.ExecuteScalar();
+                }
+            }
+
+            readonly string _name;
+            public string ConnectionString => $@"Data Source=(LocalDB)\MSSQLLocalDB;Initial Catalog={_name};Integrated Security=True;Connection Timeout=300;";
```

Comment:
isn't this the same as https://github.com/Particular/ServiceControl/pull/2886/files#diff-3bfe53cbb55e0ef07db3f50bc5e368875b362bc6a5e622c076d49889029d9dc3R192 ?